packages/wallets/src/features/cashier/components/WalletCashierHeader/WalletCashierHeader.tsx

```diff
@@ -1,84 +1,162 @@
-import React from 'react';
-import { useHistory, useLocation } from 'react-router-dom';
+import React, { useEffect, useRef } from 'react';
 import classNames from 'classnames';
+import { useHistory, useLocation } from 'react-router-dom';
 import { useActiveWalletAccount } from '@deriv/api';
-import { WalletCardIcon } from '../../../../components/WalletCardIcon';
-import { WalletGradientBackground } from '../../../../components/WalletGradientBackground';
+import { WalletCardIcon, WalletGradientBackground, WalletText } from '../../../../components';
 import { WalletListCardBadge } from '../../../../components/WalletListCardBadge';
 import useDevice from '../../../../hooks/useDevice';
 import CloseIcon from '../../../../public/images/close-icon.svg';
+import IcCashierAdd from '../../../../public/images/ic-cashier-deposit.svg';
+import IcCashierStatement from '../../../../public/images/ic-cashier-statement.svg';
+import IcCashierTransfer from '../../../../public/images/ic-cashier-transfer.svg';
+import IcCashierWithdrawal from '../../../../public/images/ic-cashier-withdrawal.svg';
+import ResetBalance from '../../../../public/images/plus-thin.svg';
 import './WalletCashierHeader.scss';
 
-const realAccountTabs = ['deposit', 'withdraw', 'transfer', 'transactions'];
-const virtualAccountTabs = ['withdraw', 'transfer', 'transactions', 'reset-balance'];
+type TProps = {
+    hideWalletDetails: boolean;
+};
+
+const realAccountTabs = [
+    {
+        icon: <IcCashierAdd />,
+        path: 'deposit',
+        text: 'Deposit',
+    },
+    {
+        icon: <IcCashierWithdrawal />,
+        path: 'withdraw',
+        text: 'Withdraw',
+    },
+    {
+        icon: <IcCashierTransfer />,
+        path: 'transfer',
+        text: 'Transfer',
+    },
+    {
+        icon: <IcCashierStatement />,
+        path: 'transactions',
+        text: 'Transactions',
+    },
+] as const;
+
+const virtualAccountTabs = [
+    {
+        icon: <IcCashierTransfer />,
+        path: 'transfer',
+        text: 'Transfer',
+    },
+    {
+        icon: <IcCashierStatement />,
+        path: 'transactions',
+        text: 'Transactions',
+    },
+    {
+        icon: <ResetBalance />,
+        path: 'reset-balance',
+        text: 'Reset balance',
+    },
+] as const;
 
-const WalletCashierHeader = ({ hideWalletDetails }: { hideWalletDetails: boolean }) => {
-    const { data } = useActiveWalletAccount();
+const WalletCashierHeader: React.FC<TProps> = ({ hideWalletDetails }) => {
+    const { data: activeWallet } = useActiveWalletAccount();
     const { isMobile } = useDevice();
+    const activeTabRef = useRef<HTMLDivElement>(null);
     const history = useHistory();
     const location = useLocation();
+    const tabs = activeWallet?.is_virtual ? virtualAccountTabs : realAccountTabs;
```

Comment:
If we switch the account from header's account switcher, will it re-render the component and change the tabs? (Just wondering)

Reply:
@farhan-nurzi-deriv 
yes it will, but i can see that if i have BTC acc and i'm on Withdrawal tab, when i switch to Demo, the screen with tabs is changed but since demo don't have withdrawal tab, we don't have an active tab. let me fix it

https://github.com/binary-com/deriv-app/assets/103181646/3cfcf426-3286-49b2-a719-1155989ee692

Reply:
@farhan-nurzi-deriv this issue will be fixed in other user story, thanks for pointing on this 🙏🏻